src/app/page.tsx

```diff
@@ -1,3 +1,403 @@
+  display: grid;
+  grid-template-columns: ${(props) => props.columnTemplate || '1fr 1fr'}; // default is two columns
+`
+const FlexContainer = styled.div<{ direction?: 'row' | 'column'; gap?: string }>`
+  display: flex;
+  flex-direction: ${(props) => props.direction || 'row'};
+  gap: ${(props) => props.gap || '0px'}
+  // height: 100%;
+  align-items: center;
+  max-width: 100vw;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+`
+
+const Text = styled.div`
+  ${FontVariant.body_md}
+  color: ${Color.gray700}
+
+  p {
+    margin: 0;
+  }
+`
+
+const SectionHeader: React.FC<{ title: string; subtitle: string }> = ({ title, subtitle }) => {
+  const Title = styled.h2`
+    ${FontVariant.title_lg}
+    display: grid;
+
+    /* Orange bar above title */
+    &:before {
+      content: '';
+      justify-self: left;
+      border: 4px solid ${Color.orange900}; // careful of margins (8px needs to be 4px here)
+      width: 48px;
+      margin-bottom: 8px;
+    }
+
+    /* Black bar underneath title */
+    // &:after {
+    //   content: '';
+    //   // justify-self: center;
+    //   border: 1px solid black;
+    //   margin-top: 8px;
+    //   width: 72px;
+    // }
+  `
+  const Subtitle = styled.h3`
+    ${FontVariant.title_sm}
+  `
+  return (
+    <div style={{ display: 'grid', gap: '8px', marginBottom: '72px' }}>
+      <Title>{title}</Title>
+      <Subtitle>{subtitle}</Subtitle>
+    </div>
+  )
+}
+
+const HeroSection = () => {
+  const RespFontSize = {
+    title_xl: '2.25rem',
+    title_lg: '1.75rem',
+    title_sm: '1rem',
+  } as const
+
+  const HeroContainer = styled.div`
+    display: flex;
+  `
+
+  const HeroTextArea = styled.div`
+    flex-basis: 50%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: start;
+    padding: 90px 96px;
+  `
+
+  const HeroTitle = styled.h1`
+    // font-size: 36px;
+    font-size: clamp(2rem, 1vw + 2rem, 3.5rem);
+    font-weight: 700;
+    letter-spacing: 0.5px;
+    text-transform: uppercase;
+    margin-bottom: 24px;
+  `
+  const HeroSubtitle = styled.h2`
+    // font-size: 22px;
+    font-size: clamp(1.25rem, 0.3125 + 4.17vw, 2.5rem);
+    font-weight: 300;
+    // line-height: 1.7rem;
+    margin-bottom: 32px;
+  `
+  const HeroMessage = styled.p`
+    // font-size: 16px;
+    font-size: clamp(1rem, 0.65vw + 0.5rem, 2rem);
+    color: ${Color.gray700};
+    text-align: justify;
+    min-width: 300px;
+    max-width: 100%;
+  `
+
+  const HeroImageContainer = styled.div`
+    flex-basis: 50%;
+    display: grid;
+    align-content: center;
+    position: relative;
+    z-index: 0;
+  `
+
+  return (
+    <Section id="hero-section" style={{ padding: '0' }}>
+      {/* padding: 0 is to allow image to stretch to the right side of the webpage*/}
+      <HeroContainer>
+        <HeroTextArea id="hero-text-area">
+          <HeroTitle>
+            Welcome to <span style={{ color: `${Color.orange900}`, paddingRight: '7px' }}>Kixlab</span>!
```

Comment:
I would just put a whitespace after KIXLAB instead of paddingRight because the padding should be responsive to the font size ideally.

Reply:
[
![Screenshot 2024-05-02 at 1 37 05 PM](https://github.com/kixlab/website/assets/85657217/26e58f3c-7f39-494a-bb61-71f13158ff9e)
](url)
If I don't use `paddingRight`, this is what the webpage will look like in certain cases.

Reply:
good point. I haven't thought of this case.